src/main/java/com/bulkbuy/enterprise/dao/OrderDAOStub.java

```diff
@@ -15,7 +15,19 @@
 
     @Override
     public Order create(Order order)  {
-        allOrders.put(order.getOrderId(),order);
+        if(order.getOrderId() <= 0){
+            List<Order> allOrderList = new ArrayList(allOrders.values());
+            int lastOrderId = 0;
+            for(int i = 0; i < allOrderList.size(); i++){
+                int indexOrderId = allOrderList.get(i).getOrderId();
+                if(indexOrderId > lastOrderId){
+                    lastOrderId = indexOrderId;
+                }
+            }
+            int tempOrderId = lastOrderId + 1;
+            order.setOrderId(tempOrderId);
+        }
+        allOrders.put(order.getOrderId(), order);
```

Comment:
Added a temporary order id generator for orders that have an id of 0 or less. This is to prevent conflicts with have the same order number for multiple orders. This is only temporary until the auto generation of the id has been implemented.